Wrappers.h

```diff
@@ -4,35 +4,35 @@
 
 struct ShoeWrapper
 {
-    ShoeWrapper(Shoe* shoePtr) : theShoe(shoePtr) { }
-    ~ShoeWrapper() { delete theShoe; }
+    ShoeWrapper(Shoe* shoePtr);
+    ~ShoeWrapper();
     Shoe* theShoe = nullptr;
 };
 
 struct WurlitzerWrapper
 {
-    WurlitzerWrapper(Wurlitzer* wurlitzerPtr) : theWurli(wurlitzerPtr) { }
-    ~WurlitzerWrapper() { delete theWurli; }
+    WurlitzerWrapper(Wurlitzer* wurlitzerPtr);
```

Comment:
You aren't using this pointer member variable in this source file so you don't need to include the header for it.  
Forward declare the type in the header.
Include in the cpp
- [ ] revise